[2025-10-20T23:36:12.156+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-20T23:36:12.188+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_Lab1_TaskFlow_FileIO.validate_loaded_task scheduled__2025-10-19T00:00:00+00:00 [queued]>
[2025-10-20T23:36:12.196+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_Lab1_TaskFlow_FileIO.validate_loaded_task scheduled__2025-10-19T00:00:00+00:00 [queued]>
[2025-10-20T23:36:12.197+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-20T23:36:12.217+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): validate_loaded_task> on 2025-10-19 00:00:00+00:00
[2025-10-20T23:36:12.244+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=125) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-20T23:36:12.255+0000] {standard_task_runner.py:63} INFO - Started process 158 to run task
[2025-10-20T23:36:12.246+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Airflow_Lab1_TaskFlow_FileIO', 'validate_loaded_task', 'scheduled__2025-10-19T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/***.py', '--cfg-path', '/tmp/tmpacnsgr28']
[2025-10-20T23:36:12.257+0000] {standard_task_runner.py:91} INFO - Job 4: Subtask validate_loaded_task
[2025-10-20T23:36:12.389+0000] {task_command.py:426} INFO - Running <TaskInstance: Airflow_Lab1_TaskFlow_FileIO.validate_loaded_task scheduled__2025-10-19T00:00:00+00:00 [running]> on host 0074cd054b76
[2025-10-20T23:36:12.658+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='your_name' AIRFLOW_CTX_DAG_ID='Airflow_Lab1_TaskFlow_FileIO' AIRFLOW_CTX_TASK_ID='validate_loaded_task' AIRFLOW_CTX_EXECUTION_DATE='2025-10-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-19T00:00:00+00:00'
[2025-10-20T23:36:12.661+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-20T23:36:12.662+0000] {airflow.py:65} INFO - Validating loaded data at /opt/***/dags/data/loaded.pkl
[2025-10-20T23:36:12.716+0000] {python.py:237} INFO - Done. Returned value was: {'rows': 8950, 'cols': 18, 'sample_head': [{'CUST_ID': 'C10001', 'BALANCE': 40.900749, 'BALANCE_FREQUENCY': 0.818182, 'PURCHASES': 95.4, 'ONEOFF_PURCHASES': 0.0, 'INSTALLMENTS_PURCHASES': 95.4, 'CASH_ADVANCE': 0.0, 'PURCHASES_FREQUENCY': 0.166667, 'ONEOFF_PURCHASES_FREQUENCY': 0.0, 'PURCHASES_INSTALLMENTS_FREQUENCY': 0.083333, 'CASH_ADVANCE_FREQUENCY': 0.0, 'CASH_ADVANCE_TRX': 0, 'PURCHASES_TRX': 2, 'CREDIT_LIMIT': 1000.0, 'PAYMENTS': 201.802084, 'MINIMUM_PAYMENTS': 139.509787, 'PRC_FULL_PAYMENT': 0.0, 'TENURE': 12}]}
[2025-10-20T23:36:12.717+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-20T23:36:12.760+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Airflow_Lab1_TaskFlow_FileIO, task_id=validate_loaded_task, run_id=scheduled__2025-10-19T00:00:00+00:00, execution_date=20251019T000000, start_date=20251020T233612, end_date=20251020T233612
[2025-10-20T23:36:12.956+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-20T23:36:12.973+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-20T23:36:12.979+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
